vocabularies/UI.xml

```diff
@@ -1605,6 +1604,16 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         <Annotation Term="Core.Description" String="The contents of this property must not be propagated to the app-to-app navigation context" />
       </Term>
 
+      <Term Name="DisplayTimeZone" Type="Edm.String" Nullable="false" AppliesTo="Property Parameter">
+        <Annotation Term="Core.Description" String="The point in time represented by the annotated property shall be presented in the context of the given time zone"/>
+        <Annotation Term="Core.LongDescription">
+          <String>The value list for time zones is not prescribed by this vocabulary.
+The `Edm.DateTimeOffset` in the annotated property SHOULD contain a [timezoneOffset](https://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#vp-dt-timezone) that corresponds to the time zone.
+In the absence of this annotation, points in time are typically presented in the current user's time zone.</String>
+        </Annotation>

```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="The point in time represented by the annotated property shall be presented in the context of the given time zone"/>
        <Annotation Term="Core.LongDescription">
          <String>The value of this annotation is only used for display and not used for timezone calculation.
Instead the [timezoneOffset](https://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#vp-dt-timezone) in the `Edm.DateTimeOffset` value of the annotated property is removed, and the dateTime part is used for displaying the timestamp. It is the service's responsibility to keep the symbolic timezone and the numeric offset in sync.</String>
        </Annotation>

```

Reply:
Simply removing the timezoneOffset may _change_ the value of the timestamp. This is justified only if it was the correct timezoneOffset to begin with. I'd rather keep my wording and possibly replace SHOULD by MUST.